src/app/helptext/network/interfaces/interfaces-form.ts

```diff
@@ -60,4 +56,65 @@ alias_netmaskbit6_tooltip : T('Select the prefix length used on the network.'),
 delete_placeholder6: T('Delete'),
 delete_tooltip6: T('Set to delete this alias.'),
 
+bridge_members_placeholder: T('Bridge Members'),
+bridge_members_tooltip: T(''),
+
+failover_critical_placeholder: T('Failover Critical'),
+failover_critical_tooltip: T(''),
```

Comment:
From TrueNAS Guide "Interface Configuration Settings" table:

"Set this interface as critical. This allows logging in to the web interface available at the Virtual IP address after a failover. Warning: At least one interface must have this option set or the web interface will become unavailable. This can also be set when configuring network interfaces in the Console Setup Menu."

Reply:
@wblock @william-gr this seems like a reasonable tooltip label to me, what do you say?

Reply:
That option more importantly means that if the interface is preempted via VRRP protocol it will cause a failover.

Reply:
"If this interface is considered critical for failover it will cause a failover event in case it is preempted by the other controller via VRRP/CARP protocol and there arent other interfaces in that same failover group."